sanic/blueprints.py

```diff
@@ -122,10 +116,10 @@ class Blueprint(BaseSanic):
     def __init__(
         self,
         name: str,
-        url_prefix: Optional[str] = None,
-        host: Optional[Union[List[str], str]] = None,
-        version: Optional[Union[int, str, float]] = None,
-        strict_slashes: Optional[bool] = None,
+        url_prefix: str | None = None,
+        host: list[str] | str | None = None,
+        version: int | (str | float) | None = None,
```

Comment:
```suggestion
        version: int | str | float | None = None,
```
Union conversions left extra parentheses in many places, fixing manually...